Shared/Samples/Show utility associations/ShowUtilityAssociationsView.swift

```diff
@@ -0,0 +1,255 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct ShowUtilityAssociationsView: View {
+    /// The view model for the sample.
+    @StateObject private var model = Model()
+    
+    /// An image that represents an attachment symbol.
+    @State var attachmentImage: Image?
+    
+    /// An image that represents a connectivity symbol.
+    @State var connectivityImage: Image?
```

Comment:
I refactored all properties onto the model besides these images since they need to be set in the model, but when updated in the model they are not updated in the toolbar where the images are displayed.